src/main/java/adeuxpas/back/dto/AdPostRequestDTO.java

```diff
@@ -0,0 +1,161 @@
+package adeuxpas.back.dto;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import adeuxpas.back.enums.AdStatus;
+
+/*
+ * This a data transfer object (DTO) that is used when receiving 
+ * an Ad object from the front 
+ * to tranform it into an AdPostDto during 
+ * the Ad creation process in order 
+ * to save it into the database
+ */
+
```

Comment:
I don't really understand why the id and the adStatus fields are included in this Request DTO. 
 
 We should not receive the id (since it's a new post, and it shouldn''t have one yet) and adStatus (we should just set this property in the service classs) in the request from the front, they're irrelevant fields here

Reply:
ok